SUI/SUI/View/LoginView.swift

```diff
@@ -0,0 +1,173 @@
+import SwiftUI
+
+struct LoginView: View {
+    private struct Constants {
+        static let login = "Log in"
+        static let signup = "Sign up"
+        static let numberPlaceholder = "+ *(***)***-**-**"
+        static let paswordPlaceholder = "Password"
+        static let eye = "eye"
+        static let eyeSlash = "eye.slash"
+        static let forgotPassword = "Forgot your password?"
+        static let verification = "Check Verification"
+        static let supportTitle = "Contact support"
+        static let contact = "Contact"
+        static let supportPhone = "+375448897634"
+    }
+
+    private enum FocusedField {
+        case phoneNumber
+        case password
+    }
+
+    var body: some View {
+        NavigationView {
+            ZStack {
+                LinearGradient.backgroundHorizontal
+                    .ignoresSafeArea()
+
+                Group {
+                    VStack {
+                        LoginSignupView
+                        Spacer()
+                            .frame(maxHeight: 76)
+                        LoginForm
+                        Spacer()
+                            .frame(maxHeight: 95)
+                        FormFooter
+                        Spacer()
+                    }
+                    .padding(.vertical, 39)
+                    .padding(.horizontal, 20)
+                }
+                .modifier(WhiteContentModifier(topOffset: 56))
+            }
+        }
+        .onTapGesture {
+            hideKeyboard()
+        }
+    }
+
+    private var LoginSignupView: some View {
+        HStack(spacing: 0) {
+            Text(Constants.login)
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+            Text(Constants.signup)
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+                .background(.bgLightGray)
+        }
+        .frame(width: 300, height: 55)
+        .font(.bold20)
+        .foregroundStyle(LinearGradient.textGradient)
+        .clipShape(Capsule())
+        .background {
+            Capsule()
+                .stroke(.bgLightGray, lineWidth: 2.0)
+        }
+    }
+
+    private var LoginForm: some View {
```

Comment:
```suggestion
    private var loginFormView: some View {
```
аналогично со всеми вью во всех файлах